typescript/src/preconditions/Spam.ts

```diff
@@ -1,20 +1,21 @@
 import { GuildSettings, readSettings, writeSettings } from '#lib/database';
 import { LanguageKeys } from '#lib/i18n/languageKeys';
+import type { GuildMessage } from '#lib/types';
+import { isModerator, isOwner } from '#utils/functions';
 import { AsyncPreconditionResult, Precondition } from '@sapphire/framework';
 import { RateLimitManager } from '@sapphire/ratelimits';
 import { Time } from '@sapphire/time-utilities';
-import type { Message } from 'discord.js';
 
 export class UserPrecondition extends Precondition {
 	private readonly ratelimit = new RateLimitManager(Time.Second * 30, 1);
 
-	public async run(message: Message): AsyncPreconditionResult {
```

Comment:
Big no no. We use the precondition in DM only command as well. Just search for `spam: true`. For example catfact and dog.

Reply:
Look at literally the line below